demo/send/campaigns/4ff6e5fe-3560-40c0-abf0-7834ed05de3d.json

```diff
@@ -0,0 +1 @@
nt-size:14px;\\\"><font color=\\\"#999999\\\">Bronx, New York</font></span></p>\\n\",\"padding\":10,\"multiPadding\":{\"allow\":false,\"paddingTop\":10,\"paddingRight\":10,\"paddingBottom\":10,\"paddingLeft\":10},\"uniqueId\":\"57b401b1-beaa-4883-85f6-a113f0c1bca2\"},{\"id\":\"c87c0014-71f8-51dc-aa76-261722549b45\",\"background_color\":\"#f5f5f5\",\"type\":\"text\",\"textBorderRadius\":0,\"textBorderWidth\":0,\"textBorderColor\":\"#000000\",\"text\":\"<p style=\\\"text-align: center;\\\"><span style=\\\"font-size:12px;\\\"><span style=\\\"color:#999999;\\\">Want to stop receiving emails from us? You can </span><a href=\\\" #unsubscribeLink#\\\" style=\\\"text-decoration:none;\\\"><span style=\\\"color:#999999;\\\"><span class=\\\"custom-style-link\\\" style=\\\"font-family: Arial, Helvetica, sans-serif;\\\"><u>unsubscribe</u></span></span></a><span style=\\\"color:#999999;\\\"> here</span></span></p>\\n\",\"padding\":10,\"multiPadding\":{\"allow\":false,\"paddingTop\":10,\"paddingRight\":10,\"paddingBottom\":10,\"paddingLeft\":10},\"uniqueId\":\"79103e8a-d468-4285-b436-448ce0484d8e\"},{\"id\":\"05b31639-6921-59db-bdeb-d6f83d4cc3e0\",\"type\":\"spacer\",\"borderWidth\":0,\"borderRadius\":0,\"borderColor\":\"#000000\",\"height\":40,\"background_color\":\"#f5f5f5\",\"uniqueId\":\"01f8be77-c8d8-4e99-9ec0-e8f9ec619d9b\"},{\"id\":\"7f0695e0-aebe-59f1-a251-4ddccd6b6478\",\"type\":\"spacer\",\"borderWidth\":0,\"borderRadius\":0,\"borderColor\":\"#000000\",\"height\":57,\"background_color\":\"transparent\",\"uniqueId\":\"18fdbda7-ee30-425d-8b2e-477c8e21bdaf\"}],\"type\":\"FULL\"}],\"originalIndex\":3,\"uniqueId\":\"54c29bbf-0f2c-464b-8747-f2dd32e2f311\"}],\"errors\":[],\"script\":{\"head\":\"// This code will be added to a <Script> tag, \\n// at the <head> of the document \\n\",\"body\":\"// This code will be added to a <Script> tag, \\n// at the end of the <body> element \\n\"},\"primaryProduct\":{\"sync\":false,\"status\":false,\"description\":\"\",\"startDate\":\"\",\"endDate\":\"\",\"couponCode\":\"\",\"src\":\"\",\"resized\":false,\"resizeWidth\":null,\"resizeHeight\":null,\"currentSrc\":\"\",\"originalSrc\":\"\"},\"watermark\":{\"style\":\"white-left\",\"alignment\":\"center\",\"padding\":20},\"templateId\":\"2caeacb0-6aab-4167-9b8d-24c77f4757b8\",\"hasRepeatables\":0,\"repeatableRowId\":\"\"}}","Plain":" SAMPLEACTIVITYHave yourself a merry little present (on us)Have yourself a merry little present (on us)This Christmas a special gift awaits you. Browse our wide range of holiday products and choose something to add to your cart. Your special freebie will be there too!This Christmas a special gift awaits you. Browse our wide range of holiday products and choose something to add to your cart. Your special freebie will be there too!Grab your present!Louder180 A32nd StreetBronx, New YorkWant to stop receiving emails from us? You can unsubscribe here "},"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":0,"MailingList":null,"Entity":null}
```

Comment:
is this "Generated for Email Activity" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?